docs/UserGuide.md

```diff
@@ -508,43 +508,40 @@ ProductiveNUS data are saved in the hard disk automatically after any command th
 3. Close the app in your other computer.
 4. Overwrite the newly created data file with the data file from your previous computer.
 5. All your existing data has been successfully transferred!
-
+<br>
 **Q**: I am not free during the time that my assignment has been scheduled. How do I reschedule?<br>
 **A**:
-You can reschedule by using the schedule command on that assignment again.
-
-1. Find out the index of the assignment that you want to reschedule.
-
-DIAGRAM
-
-2. Type the schedule command with the correct index into the command box .
-
-DIAGRAM
+<br>
```

Comment:
The formatting here appears to have a minor error as well.

![image](https://user-images.githubusercontent.com/58943071/98028139-9ae35380-1e48-11eb-9800-c693b9d7d46d.png)